services/cd-service/pkg/cmd/server.go

```diff
@@ -165,7 +167,7 @@ func RunServer() {
 			if err != nil {
 				logger.FromContext(ctx).Fatal("datadog.metrics.error", zap.Error(err))
 			}
-			ctx = context.WithValue(ctx, "ddMetrics", ddMetrics)
+			ctx = context.WithValue(ctx, "ddMetrics", ddMetrics) //nolint: staticcheck
```

Comment:
what is the issue here? Could you link to the documentation with all the different linters?

Reply:
This was the error
`SA1029: should not use built-in type string as key for value; define your own type to avoid collisions (staticcheck)`

I defined a `contextKey` type and used it to resolve the error